saos-webapp/src/main/webapp/WEB-INF/static/stylesheet/scss/newLayout.scss

```diff
@@ -841,7 +845,7 @@ a, .hint {
     
     .navigation {
     
-        input[type="text"] {
+        input[type="text"], button.input-button {
             padding-left: 35px;
             
         }
```

Comment:
I think the input-button should be similar but not identical to phrase inputs. I would make it a little more grayish. E.g. color of the border could be aaa, color of the span background 777. Additionaly, i would change 'Dodaj hasło' to '+ Dodaj hasło'.